docs/modules/ROOT/pages/sentinel-api-reference.adoc

```diff
@@ -0,0 +1,415 @@
+  autotaskTrigger?: string;
+  alertThreshold?: Threshold;
+  alertTimeoutMs?: number;
+  notificationChannels: string[];
+}
+```
+
+An example is provided below. This sentinel will be named `My New Sentinel` and will be monitoring the `renounceOwnership` function on the `0x0f06aB75c7DD497981b75CD82F6566e3a5CAd8f2` contract on the Rinkeby network.
+The alert threshold is set to 2 times within 1 hour, and the user will be notified via email.
+
+```js
+const requestParameters = {
+  network: 'rinkeby',
+  // optional
+  confirmLevel: 1, // if not set, we pick the blockwatcher for the chosen network with the lowest offset
+  name: 'My New Sentinel',
+  address: '0x0f06aB75c7DD497981b75CD82F6566e3a5CAd8f2',
+  abi: '[{"inputs":[],"stateMutability":"nonpayable","type":"constructor"},{...}]',
+  // optional
+  paused: false,
+  conditions: [
+    {
+      eventConditions: [],
+      functionConditions: [{ functionSignature: 'renounceOwnership()' }],
+    },
+  ],
+  // optional
+  txCondition: undefined,
+  // optional
+  autotaskCondition: '3dcfee82-f5bd-43e3-8480-0676e5c28964',
+  // optional
+  autotaskTrigger: undefined,
+  // optional
+  alertThreshold: {
+    amount: 2,
+    windowSeconds: 3600,
+  },
+  // optional
+  alertTimeoutMs: 0,
+  notificationChannels: [notification.notificationId],
+};
+```
+
+Once all required parameters are populated, you can create a sentinel by calling the `create` function on the client. This will return a `CreateSentinelResponse` object.
+
+```js
+await client.create(requestParameters);
+```
+
+If you wish to trigger the sentinel based on additional events, you could append another `ConditionSet` object to the list, for example:
+
+```js
+conditions: [
+  {
+    functionConditions: [{ functionSignature: 'renounceOwnership()' }],
+    eventConditions: [
+      {
+        eventSignature: "OwnershipTransferred(address,address)",
+        expression: "\"0xf5453Ac1b5A978024F0469ea36Be25887EA812b5,0x6B9501462d48F7e78Ba11c98508ee16d29a03412\""
+      }
+    ],
+  },
+],
+```
+
+You could also apply a transaction condition by modifying the `txCondition` property:
+Possible variables: `value`, `gasPrice`, `gasUsed`, `to`, `from`, `nonce`, `status` ('success', 'failed' or 'any'), `input`, or `transactionIndex`.
+
+```js
+txCondition: 'gasPrice > 0',
+```
+
+Additionally, the sentinel could invoke an autotask to further evaluate. Documentation around this can be found here: https://docs.openzeppelin.com/defender/sentinel#autotask_conditions.
+
+```js
+// If other conditions match, the sentinel will invoke this autotask to further evaluate.
+autotaskCondition: '3dcfee82-f5bd-43e3-8480-0676e5c28964',
+// Define autotask within the notification configuration
+autotaskTrigger: '1abfee11-a5bc-51e5-1180-0675a5b24c61',
+```
+
+The `subscribers` endpoint is used to a create new sentinels via a `POST` request.
+If you wish to call the API directly, you will need to construct a `CreateBlockSubscriberRequest` object.
```

Comment:
Let's add a warning that the Defender UI currently only supports a limited subset of Sentinels (only a single addressRule, etc), and we strongly suggest going through the js client to avoid incompatibilities.